metadata-ingestion/src/datahub/ingestion/api/registry.py

```diff
@@ -30,6 +32,27 @@ def is_enabled(self, key: str) -> bool:
         tp = self._mapping[key]
         return not isinstance(tp, Exception)
 
+    def load(self, path: str, cls: type) -> None:
+        for entry_point in pkg_resources.iter_entry_points(path):
```

Comment:
var naming nit - it's not a `path` since it's really an arbitrary string that we picked